### PR TITLE
Add an API to get stat var path in the hierarchhy

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: proto/mixer.proto [hunk 1]
```diff
@@ -526,6 +526,16 @@ message RelatedPlacesInfo {
   reserved 4, 5, 6;
 }
 
+message GetStatVarPathRequest {
+  // DCID of the stat var (group)
```
Comment (pradh) [inline proto/mixer.proto:530]: Is it SV or SVG? clarify...

File: proto/mixer.proto [hunk 2]
```diff
@@ -526,6 +526,16 @@ message RelatedPlacesInfo {
...
+  string id = 1;
+}
+
+message GetStatVarPathResponse {
+  // A list of DCIDs from the queried stat var to the root of stat var hierarchy.
```
Comment (pradh) [inline proto/mixer.proto:535]: Clarify perhaps that it is a single path, out of possibly multiple, to the root. And clarify the order?

File: proto/mixer.proto [hunk 3]
```diff
@@ -526,6 +526,16 @@ message RelatedPlacesInfo {
   reserved 4, 5, 6;
 }
 
+message GetStatVarPathRequest {
```
Comment (pradh) [inline proto/mixer.proto:529]: This is actually a path of SVGs, so should it be GetStatVarGroupPathRequest ?

File: proto/mixer.proto (hunk 2 above)
Comment (pradh) [inline proto/mixer.proto:535]: StatVarGroup DCIDs?

**Review by pradh — APPROVED** [review]

File: proto/mixer.proto (hunk 3 above)
Comment (shifucun) [reply proto/mixer.proto:529]: Would like to de-emphasize stat var group in API naming and URL as it's more of a way to represent a set of stat vars. From user perspective, they are interested in stat var.

File: proto/mixer.proto (hunk 2 above)
Comment (shifucun) [reply proto/mixer.proto]: Done

Comment (shifucun) [reply proto/mixer.proto]: Done

File: proto/mixer.proto (hunk 1 above)
Comment (shifucun) [reply proto/mixer.proto]: It could be both, updated

**Review by shifucun — COMMENTED** [review]
Thanks for review!

**Pull request merged**